lib/matplotlib/ticker.py

```diff
@@ -727,7 +727,10 @@ def _set_orderOfMagnitude(self, range):
                 oom = 0
             else:
                 oom = math.floor(math.log10(val))
-        if oom <= self._powerlimits[0]:
+        if self._powerlimits[0] == self._powerlimits[1] != 0:
```

Comment:
I guess this if statement could go up above to save calculating `oom`, which we don't end up using in this case...

Reply:
Thanks for the suggestion. I've moved it higher up.